spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/support/PubSubTopicUtils.java

```diff
@@ -40,7 +41,9 @@ private PubSubTopicUtils() {
 	 * @param topic the topic name in the project or the fully-qualified project name
 	 * @param projectId the project ID to use if the topic is not a fully-qualified name
 	 * @return the Pub/Sub object representing the topic name
+	 * @deprecated Use toTopicName instead.
 	 */
+	@Deprecated
 	public static ProjectTopicName toProjectTopicName(String topic, @Nullable String projectId) {
```

Comment:
Is `ProjectTopicName` really deprecated? It looks like it was at some point, but [not anymore](https://github.com/googleapis/java-pubsub/commit/bf316caa67b0e475d8d2148ea5d09d3e9cac3693).

Reply:
`ProjectTopicName` isn't - the `createTopic` method is, though: https://github.com/googleapis/java-pubsub/blob/master/google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/TopicAdminClient.java#L258

Reply:
I see. I guess you can just do `TopicName name = TopicName.ofProjectTopicName("[PROJECT]", "[TOPIC]");` as they show in the javadoc.

Reply:
I un-deprecated this method since PTN isn't deprecated.